lib/scss/stanford_events.node.scss

```diff
@@ -148,6 +157,13 @@
     }
   }
 
+  .su-event-text-past {
+    @include caption;
+
+    color: #7f7776; //$su-color-accent-stone
```

Comment:
shouldn't we use the variable instead of the hex value?

Reply:
That variable hasn't been merged into Decanter yet.